deploy/platform/kubernetes/Makefile

```diff
@@ -136,6 +136,13 @@ feature-kubernetes-monitor:
 	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set kube-state-metrics.enabled=true)
 	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set opentelemetry.enabled=true)
 
+.PHONY: feature-elasticsearch-monitor
+feature-elasticsearch-monitor:
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set features.elasticsearchMonitor.enabled=true)
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set prometheus-elasticsearch-exporter.enabled=true)
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set prometheus-elasticsearch-exporter.es.uri=http://elasticsearch-master:9200)
```

Comment:
This implicitly depends on the fact that OAP should be using ElasticSearch as storage, and if not, the deployment may crash.

Reply:
What is recommended if the Elasticsearch is not used? 
A warning or some mock Elasticsearch?

Reply:
Would this work?
![image](https://user-images.githubusercontent.com/37059529/233753044-7cdcfe9b-adc3-4398-974c-1c61f059c4da.png)

Otherwise, we will have to mock a new ES.

Reply:
Mocking a new ES will cause the ES metrics information visible in the demo not to be from OAP ES.

Reply:
So, will this new mock ES(when needed) work with a traffic generator?

Reply:
> Would this work?
> 
> ![image](https://user-images.githubusercontent.com/37059529/233753044-7cdcfe9b-adc3-4398-974c-1c61f059c4da.png)
> 
> 
> 
> Otherwise, we will have to mock a new ES.
> 
> 

This doesn't work as expected. The `elasticsearch` feature will install ES but if the user also set `postgresql` feature, which one will the OAP store data is not determined and might be unexpected. Also if the data is stored in psql the ES will have no traffic

Reply:
I thought about your concerns, and in the end I didn't do it that way. Instead, I just provided an option, but in this option, only export the ES metrics used by OAP. If  use pgsql as the storage for OAP, the ES will have no traffic.

Reply:
`
FEATURE_FLAGS=postgresql,elasticsearch
`

Configuring both PostgreSQL and Elasticsearch at the same time seems to cause confusion. Do you have any suggestions?

Reply:
I think we need some checks before the installation. Currently we have some features that cannot be enabled at the same time. For example the storage options.

Reply:
@kezhenxu94 Is this a part of this PR? I feel we don't check that for now.
Conflict check seems a new feature to add?

Reply:
I mean we need a check mechanism to check conflict features or / and feature dependencies. I can add that  mechanism and add feature conflict check for storage options. And then @innerpeacez can add feature dependencies check in this PR